docs/book/src/user/quick-start.md

```diff
@@ -1357,18 +1358,42 @@ Note: To use the default clusterctl method to retrieve kubeconfig for a workload
 {{#/tab }}
 {{#/tabs }}
 
+### Install a Cloud Provider
+
+The Kubernetes in-tree cloud provider implementations are being [removed](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers) in favor of external cloud providers (i.e. out-of-tree). To learn more, see [this blog post](https://kubernetes.io/blog/2019/04/17/the-future-of-cloud-providers-in-kubernetes/).
```

Comment:
Would be nice to add 1-2 sentences on what is a "cloud provider"?  i.e. What if I skip this step or get it wrong?